Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -28,10 +28,19 @@ public struct OfflineMapAreasView: View {
     /// A Boolean value indicating whether the preplanned map areas are being reloaded.
     @State private var isReloadingPreplannedMapAreas = false
     
+    /// The currently selected map.
+    private var selectedMap: Binding<Map?>
+    
     /// Creates an `OfflineMapAreasView` with a given web map.
```

Comment:
There really isn't a need to repeat the type in initializer descriptions.

```suggestion
    /// Creates a view with a given web map.
```

By the way, I think the doc for this initializer could be improved. Isn't the selected map a web map too? What's the difference? `online` in the name of the first parameter helps, but the doc should bring that out, explaining what "online" means. For example, "The map to be taken offline."

Reply:
The selected map is not a web map, its a map from a preplanned map area's mobile map package.

Reply:
I see. Thanks.